### DISCUSSION TITLE
Tip for the Python int type in Mojo

### DISCUSSION THREAD

**Discussion by czheo** [modular/modular#135] [Show and tell]
Though the Mojo doc has [a section](https://docs.modular.com/mojo/programming-manual.html#int-vs-int) mentioning `Int` vs `int`, there is no `int` in Mojo now to handle arbitrarily large numbers.
Here's a workaround to have `int`.

``` py
from PythonObject import PythonObject
alias int = PythonObject

let x : int = 2
print(x ** 100) # `int` does not overflow: 1267650600228229401496703205376
print(2 ** 100) # `Int` overflows: 0
```

**Comment by usamaahmed23** [comment]
Update: The module's name is `python.object` and not `PythonObject`.
